src/extensions/step_handlers/swupdate_handler_v2/src/swupdate_handler_v2.cpp

```diff
@@ -783,14 +783,26 @@ ADUC_Result SWUpdateHandlerImpl::PrepareCommandArguments(
     }
 
     // Default options.
+
+    // Note: Add '--workfolder' option here for backward compatibility.
```

Comment:
was 'workfolder' ever supported?  I think it would be better to remove entirely, unless HILO (or some other customer with a product in production) is depending on it

Reply:
Removed